src/index.js

```diff
@@ -2,24 +2,26 @@ import BootstrapCommand from "./commands/BootstrapCommand";
 import PublishCommand from "./commands/PublishCommand";
 import UpdatedCommand from "./commands/UpdatedCommand";
 import ImportCommand from "./commands/ImportCommand";
+import OwnerCommand from "./commands/OwnerCommand";
 import CleanCommand from "./commands/CleanCommand";
 import DiffCommand from "./commands/DiffCommand";
 import InitCommand from "./commands/InitCommand";
-import RunCommand from "./commands/RunCommand";
 import ExecCommand from "./commands/ExecCommand";
+import RunCommand from "./commands/RunCommand";
 import LsCommand from "./commands/LsCommand";
 
 export const __commands__ = {
   bootstrap: BootstrapCommand,
   publish: PublishCommand,
   updated: UpdatedCommand,
   import: ImportCommand,
+  owner: OwnerCommand,
   clean: CleanCommand,
   diff: DiffCommand,
   init: InitCommand,
-  run: RunCommand,
   exec: ExecCommand,
-  ls: LsCommand
+  run: RunCommand,
+  ls: LsCommand,
```

Comment:
Heads up: This is going to conflict with the [automatic hoisting PR](https://github.com/lerna/lerna/pull/507/files#diff-1fdf421c05c1140f6d71444ea2b27638).

Might help the merge to trim it down to just the add of `owner` for now and handle the reorder separately.